devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -16,7 +17,7 @@ import groovy.json.JsonOutput
 metadata {
     definition (name: "Arrival Sensor HA", namespace: "smartthings", author: "SmartThings",
             runLocally: true, minHubCoreVersion: '000.025.00032', executeCommandsLocally: true) {
-        capability "Tone"
+		capability "Tone"
```

Comment:
Please, could You fix this indentation ?

Reply:
OK! I will fix this indentation.